test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -606,8 +606,9 @@ public void TensorFlowTransformCifar()
         public void TensorFlowTransformCifarSavedModel()
         {
             var modelLocation = "cifar_saved_model";
-
             var mlContext = new MLContext(seed: 1, conc: 1);
+            var loadModelSchema = TensorFlowUtils.GetModelSchema(mlContext, modelLocation);
+            Assert.Equal(335, loadModelSchema.Count);
```

Comment:
>Assert.Equal(335, loadModelSchema.Count [](start = 12, length = 39)

am curious , what does this count signify ?

Reply:
It's all the nodes inside tensor flow graph.
I just wanted to test GetModelSchema on unfrozen model.

---
In reply to: [246930752](https://github.com/dotnet/machinelearning/pull/2112#discussion_r246930752) [](ancestors = 246930752)